### PR TITLE
[migration] migrate plugins/audio

### PR DESCRIPTION
**Pull request by psybuzz**

### CODE REVIEW

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```
Comment (stephanwlee) [inline tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts]: Although git did not think so, this is effectively a move from tf-audio-dashboard.html and we should keep the license year there. Change this to 2016.

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      (audio: TfAudioLoader) => {
+        audio.reload();
+      }
+    );
+  }
+  _shouldOpen(index: number) {
+    return index <= 2;
+  }
+  @computed('_runToTagInfo', '_selectedRuns', '_tagFilter')
+  get _categories() {
```
Comment (stephanwlee) [inline tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts]: can you do me a favor and specify the return type for this? This method is a bit more involved and I cannot picture what the return type should be.

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.ts [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+import {customElement, property, computed, observe} from '@polymer/decorators';
+import '@polymer/paper-slider';
+
+import * as tf_backend from '../../../components/tf_backend';
+import {formatDate} from '../../../components/tf_card_heading/util';
+import {runsColorScale} from '../../../components/tf_color_scale/colorScale';
+import '../../../components/tf_dashboard_common/tensorboard-color';
+import '../../../components/tf_markdown_view/tf-markdown-view';
+import {AudioBackendMetadataResponse, AudioBackendTagInfo} from './types';
+'use strict';
```
Comment (stephanwlee) [inline tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.ts]: Remove me. this is extraneous now.

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts (hunk 1 above)
Comment (psybuzz) [reply tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts]: Done

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts (hunk 2 above)
Comment (psybuzz) [reply tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.ts]: Done.  Yeah, I should have annotated this when migrating Images, too.

File: tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.ts (hunk 3 above)
Comment (psybuzz) [reply tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.ts]: Done

**Pull request merged**